lib/utils.pm

```diff
@@ -2044,4 +2045,14 @@ sub permit_root_ssh_in_sol {
     enter_cmd("[ `grep \"^PermitRootLogin *yes\" $sshd_config_file | wc -l` -gt 0 ] || (echo 'PermitRootLogin yes' >>$sshd_config_file; systemctl restart sshd)");
 }
 
+sub cleanup_disk_space {
```

Comment:
I think you should add some comment to explain how to clean up disk space. It is very useful for others to use this function.

Reply:
OK.